DisCatSharp.Docs/articles/modules/audio/lavalink_v4/commands.md

```diff
@@ -96,7 +96,7 @@ public class MyFirstLavalinkCommands : BaseCommandModule
 	}
 
 	[SlashCommand("leave", "Leave the voice channel")]
-	public async Task LeaveAsync(InteractionContext ctx)
+	public async Task LeaveAsync(InteractionContext ctx, DiscordChannel channel)
```

Comment:
This parameter would go unused.

Reply:
Nvm, it wouldn't. I missed the `channel.Mention` at the bottom. This needs a different change than adding a parameter though.

It should probably store the channel its about to leave in a variable instead of adding a parameter.

Reply:
> Nvm, it wouldn't. I missed the `channel.Mention` at the bottom. This needs a different change than adding a parameter though.
> 
> It should probably store the channel its about to leave in a variable instead of adding a parameter.

So like adding a variable inside like `var channel = ctx.Channel as DiscordChannel;` instead of putting it in the parameter?
`

Reply:
Not necessarily ctx.Channel, that's the channel the command is ran in. I'm not fully sure what the exact variable its called but im pretty sure theres some channel variable on the guild player. And also the `as DiscordChannel` wont be necessary.

But instead of this whole hassle, i think we can just remove the channel mention from the message and make it say something along the lines of "left channel" instead. Up to you which route to take.

Reply:
Hmm yeah I guess, in my use case I just added the parameter like in the join command above it.

Reply:
u can access the current channel before disconnecting through guildPlayer.Channel. So you don't need the argument.

Reply:
Oh yeah, `guildPlayer.Channel.Mention` seems to work just fine so I guess we don't need `DiscordChannel channel` in the parameter, thanks